### PR TITLE
Update mir-core version

### PR DESCRIPTION
**Pull request by 9il**

### CODE REVIEW

File: dub.sdl [hunk 1]
```diff
@@ -3,4 +3,4 @@ description "Extracted std.experimental.allocator"
 authors "Team Phobos"
 copyright "Copyright © 2017, Team Phobos"
 license "BSL-1.0"
-dependency "mir-core" version="~>0.0.5"
+dependency "mir-core" version=">0.0.5 <0.3.0"
```
Comment (wilzbach) [inline dub.sdl]: why < 0.3.0?

Comment (9il) [reply dub.sdl]: The current version is v0.2.0, `< 0.3.0` includes all v0.2.x, and not include 0.3.0

Comment (wilzbach) [reply dub.sdl]: Yes, but so would `~>0.2.0`.

https://dub.pm/package-format-sdl.html#version-specs

Comment (9il) [reply dub.sdl]: The current change is `>0.0.5 <0.3.0` the `~>0.2.0` is different and does not include older versions. BTW, it should be `>=0.0.5 <0.3.0` to support older configs, will change it

**Pull request merged**